### PR TITLE
AQSG support

### PR DESCRIPTION
**Pull request by larshp**
test repository = https://github.com/abapGit-tests/AQSG

looks like it doesnt require any rules, or has the changed by user

### CODE REVIEW

File: src/objects/zcl_abapgit_object_aqsg.clas.abap [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_object~get_comparator.
+    RETURN.
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_object~get_deserialize_steps.
+    APPEND zif_abapgit_object=>gc_step_id-late TO rt_steps.
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_aqsg.clas.abap:86]: any reason why this is not `abap`?

**Review by mbtools — COMMENTED** [review]
PS: My comments also apply to `aqbg` which I hadn't looked at closely.

File: src/objects/zcl_abapgit_object_aqsg.clas.abap (hunk 1 above)
Comment (larshp) [reply src/objects/zcl_abapgit_object_aqsg.clas.abap:86]: I dont think anything can depend on ABAP query stuff, so moving it to late. I think ABAP Query can reuse ABAP code, but not sure

"Used for objects that depend on other objects processed in the previous two phases"

File: src/objects/zcl_abapgit_object_aqsg.clas.abap [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+  METHOD zif_abapgit_object~changed_by.
+    rv_user = zcl_abapgit_objects_super=>c_user_unknown.
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_object~delete.
+
+    set_default_transport( iv_transport ).
```
[the remark this answers is not in the document]
Comment (larshp) [reply src/objects/zcl_abapgit_object_aqsg.clas.abap]: removed, also for AQBG

File: src/objects/zcl_abapgit_object_aqsg.clas.abap (hunk 1 above)
Comment (mbtools) [reply src/objects/zcl_abapgit_object_aqsg.clas.abap:82]: 👍

**Review by mbtools — APPROVED** [review]
👍

**Pull request merged**